main.qml

```diff
@@ -1363,8 +1363,9 @@ ApplicationWindow {
             return "";
         }
 
-        property string language
-        property string locale
+        property string language : 'English (US)'
```

Comment:
Maybe drop the space preceding the colon

Reply:
Ah, i see, there are other properties declared this way. But still think we usually don't have the spacing here.